runner/context.ts

```diff
@@ -18,12 +18,17 @@ import {BrowserRunner} from './browserrunner';
 import * as config from './config';
 import {Plugin} from './plugin';
 
-type Handler = (o: {}, callback: (err: any) => void) => void;
+type Handler = (o: {}) => Promise<any>;
 
 /**
  * Exposes the current state of a WCT run, and emits events/hooks for anyone
  * downstream to listen to.
  *
+ * TODO(rictic): break back-compat with plugins by moving hooks entirely away
```

Comment:
There do! https://www.npmjs.com/search?q=wct and https://www.npmjs.com/search?q=web-component-tester show a bunch. Google search ranking suggests that this code coverage plugin is the most popular: https://www.npmjs.com/package/web-component-tester-istanbul